### PR TITLE
Replace websocket with http calls

### PR DESCRIPTION
**Pull request by m-cattena**
Post each event separately

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "actions-recorder",
-  "version": "0.0.13",
+  "version": "0.0.14",
   "description": "",
   "private": true,
   "scripts": {
-    "build": "npx webpack --config webpack.config.js --mode=production --env.RECORDER_URL='wss://recorder.testrigor.com'",
+    "build": "npx webpack --config webpack.config.js --mode=production --env.RECORDER_URL='http://recorder.testrigor.com'",
```
Comment (parunruaivo) [inline package.json]: Needs to be https://

File: src/recorder/recorder.js [hunk 2]
```diff
@@ -17,26 +16,30 @@ export default class Recorder {
     this.eventListener = new EventListener(config);
     if (Recorder.shouldConnectWs(config)) {
```
Comment (parunruaivo) [inline src/recorder/recorder.js]: Rename it ofr something like `shouldDispatchEvents`

File: src/recorder/recorder.js [hunk 3]
```diff
@@ -17,26 +16,30 @@ export default class Recorder {
...
   }
 
-  subscribeToEvents() {
-    this.eventListener
-      .events()
-      .subscribe((event) => {
-        this.webSocket.next(event);
-      });
+  postNewEvent(event) {
+    ajax({
```
Comment (parunruaivo) [inline src/recorder/recorder.js]: If there is an error, should we catch it and avoid throwing to the client?

Comment (m-cattena) [reply src/recorder/recorder.js]: yeah I was thinking about that, they are going to flood the users' consoles if we don't catch them

**Pull request merged**